api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -26,6 +25,20 @@
     <handler file="server_unavailable.html" />
   </static-error-handlers>
 
+  <static-files>
+    <include path="static/*">
```

Comment:
Can you verify that this is picking up aou-snippets-menu.js? Pretty sure you need `**` to pickup subdirs. Alternative fix would be to just to not mirror the generated/ dir there, just copy it straight into /static. That seems fine, if not correct to me, since all of these static files are copied in during the build time anyways - the fact that this one file is generated is no longer relevant.

Reply:
You're right, it is not making it. I will ** it anyway, since we should allow subdirectories, but also get rid of the generated subdir